scripts/ci/dockerfiles/malus_injected.Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN apt-get update && \
 
 
 # add adder-collator binary to docker image
-COPY ./malus /usr/local/bin
+COPY ./malus ./polkadot-*-worker /usr/local/bin
```

Comment:
I'm not sure if bash-style globbing is supported in dockerfile, if yes, you can be more explicit like `polkadot-{prepare,execute}-worker`. But I didn't want to specify full filenames because, in that case, Docker would treat missing files as an error (afaik having a glob producing zero matched is okay for Docker, but missing file is not). And I wanted to preserve backward compatibility, that is, if files are missing, just don't copy them. I'm not sure at all if it makes any sense to preserve compatibility here as CI is versioned as well as the code itself.

Reply:
Got it. I'll try without the glob and see if anything blows up.